util/http.js

```diff
@@ -28,7 +28,11 @@ exports.get = function (options, resolve, reject) {
     });
     response.on('end', function () {
       if (fmt !== 'html') {
-        body = JSON.parse(body);
+        try {
+          body = JSON.parse(body);
+        } catch (e) {
+          reject(e);
```

Comment:
Can you please add a `return` after this line to prevent the `resolve` from being called afterward?